spec/src/main/asciidoc/method-query.asciidoc

```diff
@@ -219,6 +218,20 @@ Jakarta Data implementations must support the following list of Query by Method
 The "Not Required For" column indicates the database types for which the respective keyword is not required or applicable.
 ====
 
+Most _Query by Method Name_ conditions require a single repository method parameter. The `Between` condition requires two parameters. `Null`, `True`, and `False` require none. The repository method parameters used for _Query by Method Name_ conditions follow the order in which the _Query by Method Name_ conditions appear within the method name.
+
+In the following example the value of the first parameter, `namePattern`, is used for `NameLike`, the values of the second and third parameters, `minYear` and `maxYear`, are used for `YearMadeBetween`, and the value of the fourth parameter, `maxPrice`, is used for `PriceLessThan`.
+
+[source,java]
+----
+List<Product> findByNameLikeAndYearMadeBetweenAndPriceLessThan(String namePattern,
+                                                               int minYear,
+                                                               int maxYear,
+                                                               float maxPrice,
+                                                               Limit limit,
+                                                               Sort<?>... sorts)
```

Comment:
```suggestion
                                                               Sort<Product>... sorts)
```

Reply:
Variable arguments with a type variable will generate a warning. It will need to be switched to Order, and I'll need to do another commit for that...

Reply:
@otaviojava  added in https://github.com/jakartaee/data/pull/635/commits/344307962066d72b8a7f1a14dcd1a016110694bb